### DISCUSSION TITLE
How to disable automatic token reset?

### DISCUSSION THREAD

**Discussion by Nurislom373** [vladimir-bukhtoyarov/bucket4j#368] [Q&A]
Hello. How to disable automatic token reset in latest version of Bucket4j?

**Comment by vladimir-bukhtoyarov** [comment]
Hello @Nurislom373 

What do you mean as token reset? Are you looking [something like this](https://github.com/bucket4j/bucket4j/discussions/303#discussioncomment-3678230)?
